sycl/test/graph/graph-explicit-dotp.cpp

```diff
@@ -80,7 +80,7 @@ int main() {
 
   auto exec_graph = g.finalize(q.get_context());
 
-  q.submit(exec_graph).wait();
+  q.submit([&](sycl::handler &h) { h.exec_graph(exec_graph); }).wait();
```

Comment:
We might want to rename the variable name from `exec_graph` to something else. Since that's the function name now.
```suggestion
  // Using shortcut for executing a graph of commands
  q.exec_graph(exec_graph).wait();
```

Reply:
good shout, I've updated all those variables to `executable_graph` to avoid the conflict